integration/test/performancetest/transmitter.go

```diff
@@ -4,16 +4,29 @@ import (
 	"context"
 	"errors"
 	"fmt"
-	"log"
+	"os"
 	"time"
-
+	"strings"
+	"log"
+	"math/rand"
 	"github.com/aws/aws-sdk-go-v2/aws"
 	"github.com/aws/aws-sdk-go-v2/config"
 	"github.com/aws/aws-sdk-go-v2/feature/dynamodb/attributevalue"
 	"github.com/aws/aws-sdk-go-v2/service/dynamodb"
 	"github.com/aws/aws-sdk-go-v2/service/dynamodb/types"
+
 )
 
+
+const (
+	UPDATE_DELAY_THRESHOLD = 60// this is how long we want to wait for random sleep in seconds
+	MAX_ATTEMPTS = 5 // number of attemps before we stop retrying to update
+	/*
+	!Warning: if this value is less than 25 there is a risk of testCases being lost.
+	This will only happen if all test threads and at the same time and get the same 
+	sleep value after first attempt to add ITEM
+	*/
```

Comment:
This comment block should be moved **above** `UPDATE_DELAY_THRESHOLD`. When I read this, I don't know what it refers to.